README.md

```diff
@@ -24,9 +24,9 @@ Please also be aware that this is an unoffical project worked on in my (Half-Sho
 ### Setup the bridge
 
 * Run ``npm install`` to grab the dependencies.
-* Run ``npm build`` to build the typescript.
+* Run ``npm run-script build`` to build the typescript.
 * Copy ``config/config.sample.yaml`` to ``config.yaml`` and edit it to reflect your setup.
-* Run ``node build/discordas.js -r -u "http://localhost:9005" -c config.yaml``
+* Run ``node build/src/discordas.js -r -u "http://localhost:9005/" -c config.yaml``
```

Comment:
@Half-Shot Did something else change that allows the `/` to come back in this step? This caused issues previously.